src/components/PeriodicTable.jsx

```diff
@@ -0,0 +1,189 @@
+import React, { useEffect, useRef } from "react"
+import PropTypes from "prop-types"
+
+import "./css/periodic-table.css"
+
+import { makeStyles, Paper } from "@material-ui/core"
+import PanZoom from "react-easy-panzoom"
+
+import { H2, H5, H6, P, P2 } from "./EasyText"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['P2' is defined but never used. (no-unused-vars)](https://app.codacy.com/app/davwheat/GCSE-Revision-Website/pullRequest?prid=4474002)